apps/chat/src/backend/serverless/template.yaml

```diff
@@ -74,6 +74,95 @@ Resources:
       Properties: 
         ServiceToken: !GetAtt ChimeAppInstanceLambda.Arn
 
+  #Lambda that creates AWS Chime App admin user
+  ChimeAppAdminLambda:
+      DependsOn: TriggerChimeAppInstanceLambda
+      Type: "AWS::Lambda::Function"
+      Properties:
+        Handler: "index.handler"
+        Role: !GetAtt LambdaExecuteRole.Arn
+        Runtime: "nodejs12.x"
+        Timeout: 60
+        Layers: 
+          - !Ref AWSSDKChimeLayer
+        Code:
+          ZipFile: >

```

Comment:
The issue with moving the lambda source code to a separate file is that there is no automatic way (besides switching to CDK) to upload the lamdba sources to S3 for CF to use. Until we migrate this to CDK, by keeping all the lambda code in the CF template, it will always be up to date.